### PR TITLE
Responsive website attempt ii

### PR DESCRIPTION
**Pull request by AnnaDodge**
I thiiiink it works now... Hopefully my testing was good enough this time! What do you think @hlwill?

### CODE REVIEW

**Comment by AnnaDodge** [comment]
On mobile the first menu click doesn't take you to the right place, but after that it seems to work. I found this problem in Safari, Chrome & Firefox - I can't work out why it does this.

**Comment by hlwill** [comment]
I've just been trying this in Firefox's Responsive Design View, it's sizing the text, but doesn't seem to collapse the menu at all any more?

File: index.html [hunk 1]
```diff
@@ -1,121 +1,174 @@
...
+		<p>
+  			<ul>
+  				<li> Is cheap and easy to use.</li>
+  				<li>Runs without internet or mains electricity.</li>
+  				<li>Fits into pre-existing infrastructure.</li>
+  				<li>Protects data by design.</li>
+  			</ul>
+			Need support? We can provide the technical expertise and training to get the system up and running in a community so information can be transmitted in seconds, rather than months.
+		</p>
+		<img class="img-in-text img-responsive" src="images/Oonagh_Helen_props.JPG" alt="Oonagh & Helen">
```
Comment (hlwill) [inline index.html:120]: Should this `&` be a `\amp` ?

**Comment by hlwill** [comment]
I don't quite know exactly what you want the website to do beside behave differently for different size screens. Should it just shrink the font? Should the menu change at a certain width?

**Comment by AnnaDodge** [comment]
I've just checked it in Firefox's Responsive View and the menu collapse is working for me. I'm beginning to think I'll never know how or why browsers behave the way they do...
Yes, I just wanted it to look better on mobile screens - all it should do is change the text margins and condense the menu.

File: index.html (hunk 1 above)
Comment (AnnaDodge) [inline index.html:120]: I don't think so, as it's in quotation marks?

Comment (hlwill) [inline index.html:120]: Yup. I think you're right. I just got scared by github's higlighting it red, all good

**Comment by hlwill** [comment]
Web browsers are awful. :sob:

**Comment by hlwill** [comment]
I'm reckoning it's probably fine to merge. It would be neater to squash some of those commits or something, but I'm happy as is :+1:

**Comment by AnnaDodge** [comment]
Great, thank you. :)

**Comment by AnnaDodge** [comment]
I keep accidentally hitting close and comment instead of just comment grrrr

**Pull request merged**